recommender/engine/reward_mapping.py

```diff
@@ -13,5 +37,12 @@ def ua_to_reward_id(_user_action):
 
     For now it just return generic reward id.
     """
+    transition_rewards_df = pd.read_csv(TRANSITION_REWARDS_CSV_PATH, index_col="source")
+
+    source = _to_abstract_page_id(user_action.source.page_id)
+    if user_action.action.order:
+        target = "order"
+    else:
+        target = _to_abstract_page_id(user_action.target.page_id)
 
-    return "generic_reward_id"
+    return transition_rewards_df.loc[source, target]
```

Comment:
assign output value to some variable (e.g. `reward_id`) and then return it. It will allow you to write good docstring with
```python
"""
Returns:
    reward_id: some description
"""
```

Whole docstring of this function should be refactored to match [google docstring standard](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). Also return type hint should be added.